src/dart-2.0.md

```diff
@@ -104,12 +111,13 @@ As the example shows, you can use the static `dynamic` type
 to indicate that the runtime type is unknown.
 One big difference between optionally typed Dart and Dart 2.0
 is that, in the former, the analyzer infers the `dynamic` type.
-In Dart 2.0, both the analyzer and
-the runtime try to infer something more specific than `dynamic`.
+In Dart 2.0, tools like the analyzer can infer
+more specific types than `dynamic`,
```

Comment:
I'd suggest a slight rewording because "more specific types than dynamic" doesn't quite read right for me.

> ... can [something-goes-here] infer types more specific than `dynamic`

Candidates for something-goes-here:
- often
- generally
- more easily